src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+
+import './HandicapLayerInfo.scss';
+
+const propTypes = {
+  t: PropTypes.func.isRequired,
+  language: PropTypes.string.isRequired,
+  properties: PropTypes.object.isRequired,
+};
+
+const defaultProps = {};
+
+const names = {
+  barrierfree: 'Barrierefreie Bahnhöfe',
```

Comment:
```suggestion
  barrierfree: 'ch.sbb.barrierfreierbahnhoefe',
```
So, we avoid redundant translations